projects/mysadna/src/app/components/project-details/project-details.component.html

```diff
@@ -1,29 +1,18 @@
-<div *ngIf="project" class="project-details">
-  <h1 class="project-name">{{project.getProjectName()}}</h1>
-  <h3 class="project-description">{{project.getProjectDescription()}}</h3>
-  <a href="{{project.getProjectSite()}}" target="blank" class="project-link">
-    <h3>
+<div class="project-details media" *ngIf="project">
+  <div class="project-name">{{project.getProjectName()}}</div>
```

Comment:
I prefer to not use h tag, because the tag is basic `div`, but with a lot of excess styles.

Reply:
Noted.